opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -181,20 +181,40 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
                             .addListener(new AsyncListener() {
                         @Override
                         public void onComplete(AsyncEvent event) throws IOException {
-                            try (Scope asyncScope = tracer.scopeManager().activate(scope.span(), true)) {
```

Comment:
I have removed activation for decorator, span is passed there explicitly. We don't need to create additional objects